meinberlin/apps/plans/models.py

```diff
@@ -129,7 +129,14 @@ def administrative_district(self):
     def topic_names(self):
         if hasattr(settings, 'A4_PROJECT_TOPICS'):
             choices = dict(settings.A4_PROJECT_TOPICS)
-            return [choices[topic] for topic in self.topics]
+            names = []
+            for topic in self.topics:
+                try:
+                    name = choices[topic]
+                except:
+                    name = topic
```

Comment:
or pass?

Reply:
Hmm. I like this one!

Reply:
Oh right! The linter complains and is absolutely right! You shouldn't just use try without the actual error. So, could you please add that?